controllers/commentsController.js

```diff
@@ -33,18 +31,80 @@ const getByCity = async (req, res) => {
         },
     });
     if (isEmpty(cities)) {
-        res.status(500)
-        return res;
+        return res.status(500);
+    }
+    return res.status(200).json({
+        main: {
+            cityName,
+            comments: cities,
+        },
+    });
+};
+
+const addComment = async (req, res) => {
+    try {
+        if (isEmpty(req.body)) {
+            return res.status(500).json({
+                error: 'Request body is empty.',
+            });
+        }
+        if (req.get('API-Key') !== process.env.HEADER_KEY) {
+            return res.status(403).json({
+                error: 'Api key is required for this function.',
+            });
+        }
+        const { cityName, commentText } = req.body;
+        const commentToSave = allowedCitiesArr.find((city) => city === cityName);
```

Comment:
fot what reason do you need this variable ?
you got cities from db, find related city to comment, and ... just create new line in db
you shoud remove 'if' or make'update'. not delete